### PR TITLE
Removing style that makes editor scroll to the top on iOS without any action from the user

### PR DESCRIPTION
**Pull request by lpagliari**
tl;dr
- avoid focus to be moved to the top every second when on iOS;
- but move back the issue that shows toolbar in the middle of the screen when editing on iOS:
image

Long story:

This PR reverts part of #2721, because that change made scroll on iOS devices very unstable. Using `position:scroll` as it was before this PR made the editor scroll to the top every second or so, even without any action from the user. I'm not sure if this issue is related to #2224, so maybe fixing one might fix both.

Anyway, I thought it was much worse to keep moving the focus to the top of the editor than having a toolbar "flying" on the screen when editing the pad.

Thoughts about these issues? Any suggestion?

### CODE REVIEW

**Comment by lpagliari** [comment]
This is a draft of the code I was using to avoid the "flying" toolbar on iOS. It was basically changing toolbar menus to `position: absolute` instead of `fixed` and moving the menu_right to the far bottom of the editor when virtual keyboard is supposed to be displayed (in this example, when an input or textarea has focus). The code is faaaaaar away from being perfect, it was only a partial hack to fix the issue for the most relevant scenario on the Etherpad instance I am using.

If anyone has any other suggestion I would be glad to give a try.

Here's the code:
```
var fixFlyingToobarOnIOS = function() {
  if (browser.ios) {
    var placeMenuRightOnBottom = $(".toolbar ul.menu_right").css('bottom') !== "auto";

    getNewCommentContainer().find('input, textarea')
    .on("focus", function() {
      $(".toolbar ul.menu_left").css("position", "absolute");
      $(".toolbar ul.menu_right").css("position", "absolute");

      if (placeMenuRightOnBottom) $(".toolbar ul.menu_right").css("top", $(document).outerHeight());
    })
    .on("blur", function() {
      $(".toolbar ul.menu_left").css("position", "");
      $(".toolbar ul.menu_right").css("position", "").css("top", "");

      if (placeMenuRightOnBottom) $(".toolbar ul.menu_right").css("top", "");
    });

    // When user changes orientation, we need to re-position menu_right
    if (placeMenuRightOnBottom) {
      var resizeTimer;
      $(window).on("resize", function() {
        clearTimeout(resizeTimer);
        resizeTimer = setTimeout(function() {
          var needToUpdateTop = $(".toolbar ul.menu_right").css("top") !== "";
          if (needToUpdateTop) $(".toolbar ul.menu_right").css("top", $(document).outerHeight());
        }, 500); // give enough time for the UI to change and editor be re-displayed
      });
    }

  }
```

**Comment by Gared** [comment]
I have no iOS device right now. So I can only merge your pull request if you think this is good to go

**Comment by lpagliari** [comment]
@Gared I've checked on my iPhone and iOS simulator, the problem is gone after removing `overflow: scroll;`.
